### PR TITLE
Move EIP-4987 to Review

### PR DESCRIPTION
**Pull request by devinaconley**
This moves EIP-4987 into `Review` and also cleans up the example assets.

See ongoing discussion here
https://ethereum-magicians.org/t/erc-standard-for-held-non-fungible-token-nfts-defi/7117

### CODE REVIEW

**Comment by Pandapip1** [comment]
Please fix the `eipw` errors (they can be viewed in the files pane).

**Comment by devinaconley** [comment]
fixed eipw issues, thanks @Pandapip1

File: assets/eip-4987/Consumer.sol [hunk 1]
```diff
@@ -3,7 +3,7 @@ Consumer
 
 https://github.com/devinaconley/token-hold-example
```
Comment (Pandapip1) [inline assets/eip-4987/Consumer.sol:4]:
```suggestion
```

File: assets/eip-4987/IERC1155Holder.sol [hunk 2]
```diff
@@ -3,7 +3,7 @@ IERC1155Holder
 
 https://github.com/devinaconley/token-hold-example
```
Comment (Pandapip1) [inline assets/eip-4987/IERC1155Holder.sol:4]:
```suggestion
```

File: assets/eip-4987/IERC20Holder.sol [hunk 3]
```diff
@@ -3,7 +3,7 @@ IERC20Holder
 
 https://github.com/devinaconley/token-hold-example
```
Comment (Pandapip1) [inline assets/eip-4987/IERC20Holder.sol:4]:
```suggestion
```

File: assets/eip-4987/IERC721Holder.sol [hunk 4]
```diff
@@ -3,7 +3,7 @@ IERC721Holder
 
 https://github.com/devinaconley/token-hold-example
```
Comment (Pandapip1) [inline assets/eip-4987/IERC721Holder.sol:4]:
```suggestion
```

File: assets/eip-4987/Vault.sol [hunk 5]
```diff
@@ -3,7 +3,7 @@ Vault
 
 https://github.com/devinaconley/token-hold-example
```
Comment (Pandapip1) [inline assets/eip-4987/Vault.sol:4]:
```suggestion
```

**Review by Pandapip1 — COMMENTED** [review]
LGTM except for this

**Comment by devinaconley** [comment]
Thanks for the look @Pandapip1, don't you think it's help to leave a reference to the full functional example?

**Comment by Pandapip1** [comment]
> Thanks for the look @Pandapip1, don't you think it's helpful to leave a reference to the full functional example?

We can't know that the contents of the URL will remain the same, and since the content of EIPs should only ever be changed when a PR to make a change is submitted, external links are prohibited. A good example of this is the `eip-4524` reference implementation.

So please do remove the external links.

**Comment by devinaconley** [comment]
Makes sense, just committed those suggestions :+1:
> We can't know that the contents of the URL will remain the same, and since the content of EIPs should only ever be changed when a PR to make a change is submitted, external links are prohibited. A good example of why is the (lack of) `eip-4524` reference implementation.

Thanks @Pandapip1

**Review by Pandapip1 — APPROVED** [review]
LGTM. Please close and re-open this pull request, or submit an empty commit.

**Comment by MicahZoltu** [comment]
@Pandapip1 I believe this class of problems is why we had Kodiak.  We should probably either fix them, or add Kodiak back.

**Comment by Pandapip1** [comment]
> @Pandapip1 I believe this class of problems is why we had Kodiak. We should probably either fix them, or add Kodiak back.

It's actually because the re-run @eth-bot workflow has broken.

**Comment by MicahZoltu** [comment]
> It's actually because the re-run @eth-bot workflow has broken. Auto-merge was enabled, but @eth-bot was still failing because it didn't re-trigger.

Any ideas how to fix that?

**Comment by Pandapip1** [comment]
A few. Now that the first PR I submitted for EIP-Bot was merged, I can start work on a solution,

**Pull request merged**